rollup.config.js

```diff
@@ -48,6 +48,11 @@ const getBabelOptions = ({useESModules}) => ({
 })
 
 const commonjsOptions = {
+  include: [
+    /\/node_modules\/react\//,
```

Comment:
This here creates the issue with flow.

Reply:
@TrySound @kof

Reply:
I get it. We use ancient rollup version which silently swallowed commonjs errors without emitting files.

Reply:
Upgrading it in separate branch

Reply:
So upgrading to rollup v1 would solve the issue?

Reply:
It will show the real error which is swallowed now.

Reply:
Seems like the error was in missed `react-display-name` package which is a transitive dependency of `theming`.